src/relay/blockchain/events_informations.py

```diff
@@ -74,7 +74,9 @@ def get_list_of_paid_interests_for_trustline_in_between_timestamps(
 
     def get_transfer_details(self, tx_hash):
 
-        all_events_of_tx = self.events_proxy.get_transaction_events(tx_hash)
+        all_events_of_tx = self.events_proxy.get_transaction_events(
+            tx_hash, standard_event_types=(TransferEventType, BalanceUpdateEventType)
```

Comment:
why are they called `standard_event_types` and not just `event_types`?

In the other parts of the codes it was only named that way, because it was the default selected event types (mmh bettter name would have been maybe `default_event_types`) if you would not chose any, but here you chose.